app/src/main/java/ar/com/wolox/android/cookbook/recipepicker/RecipePickerFragment.kt

```diff
@@ -1,27 +1,37 @@
 package ar.com.wolox.android.cookbook.recipepicker
 
-import android.os.Handler
-import android.os.Looper
 import ar.com.wolox.android.cookbook.R
 import ar.com.wolox.wolmo.core.fragment.WolmoFragment
-import ar.com.wolox.wolmo.core.util.ToastFactory
-import javax.inject.Inject
+import kotlinx.android.synthetic.main.fragment_recipe_picker.*
 
-class RecipePickerFragment : WolmoFragment<RecipePickerPresenter>() {
+class RecipePickerFragment : WolmoFragment<RecipePickerPresenter>(), RecipePickerView {
 
-    @Inject lateinit var toastFactory: ToastFactory
+    override fun layout() = R.layout.fragment_recipe_picker
 
-    override fun layout() = R.layout.fragment_base
+    override fun init() {}
 
-    override fun init() {
-        Handler(Looper.getMainLooper()).postDelayed({
-                    toastFactory.show("Welcome to Wolmo Android Cookbook!")
-                },
-                1500)
+    override fun showRecipes(recipes: List<Recipe>) {
+        vRecipePickerSelectionViewPager.apply {
+            adapter = RecipeViewPager(mapRecipesToItems(recipes)) {
+                presenter.onRecipeCLicked(it)
+            }
+            setPageTransformer(false, CarouselEffectTransformer())
+            pageMargin = resources.getDimensionPixelSize(R.dimen.spacing_medium_more)
+        }
     }
 
-    companion object {
+    private fun mapRecipesToItems(recipes: List<Recipe>): List<RecipeItem> {
```

Comment:
We should add some kind of instruction to add a new recipe here. Maybe some documentation here.